code/modules/events/wormholes.dm

```diff
@@ -22,7 +22,8 @@
 
 /datum/round_event/wormholes/start()
 	for(var/turf/open/floor/T in world)
-		if(is_station_level(T.z))
+		var/area/A = get_area(T)
+		if(is_station_level(T.z) && !A.outdoors)
```

Comment:
```suggestion
		if(is_station_level(T.z))
				var/area/A = get_area(T)
				if(!A.outdoors)
					continue
```
This way we only get the area after confirming the turf is on station level for performance, since this is a world loop after all.

Reply:
Thanks fam